packages/admin-panel/src/pages/resources/EntitiesPage.js

```diff
@@ -98,10 +57,13 @@ const FIELDS = [
     },
   },
   {
-    Header: 'QR Code',
+    Header: 'QR',
     source: 'id',
-    type: 'logs',
-    Cell: QRCodeButton,
+    type: 'qrCode',
```

Comment:
Good idea to go with this approach since it's consistent with our other existing modals, but isn't it such an awkward pattern? The logic ends up spread over a bunch of locations, and the modal exists on every page. Maybe we should think about refactoring this part of the Admin Panel otherwise it could grow out of hand a bit...

Reply:
Yeah it's not ideal but we don't really have the time to do the refactor in this project, maybe the next one.

Reply:
Oh yup, agreed 👍